tests/test_dse.py

```diff
@@ -1626,7 +1627,7 @@ def test_hoisting_symbolic_divs(self):
         op = Operator(eq)
 
         assert op._profiler._sections['section0'].sops == 1
-        assert op.body.body[1].body[0].body[0].expr.rhs == s0**-s1
```

Comment:
why this? I suppose with Arm a slightly different code gets generated... is it the lack of the denormals flag?

Reply:
Yes the [1] assumes there is the denormal body first so breaks on non-sse